docs/organizations/using-a-coding-standard.md

```diff
@@ -0,0 +1,95 @@
+# Using a coding standard
+
+Create a coding standard on your organization to define and apply global tool and code pattern configurations consistently across your repositories.
```

Comment:
I have the same feeling here with "global" as a term for this. What about "common" or "shared"?

Reply:
I understand that "global" is too broad. I'll go with "shared" instead! :+1: